openapi/__tests__/definitions.test.ts

```diff
@@ -232,6 +235,50 @@ const aPayee: Payee = { fiscal_code: anOrganizationFiscalCode };
 
 const aThirdPartyId = "aThirdPartyId" as NonEmptyString;
 
+describe("BaseEnrichedMessage definition", () => {
+  it("should decode a BaseEnrichedMessage", () => {
```

Comment:
I'd add a test to check that decoding an `EnrichedMessage` sending also an `organization_fiscal_code` will not break. Because if the backend is using `EnrichedMessage` and we deploy the new functions sending an `EnrichedMessageWithOrganizationFiscalCode` we are gonna break it.

Reply:
[b0adfd3](https://github.com/pagopa/io-functions-commons/pull/379/commits/b0adfd37832213d4cb08fda8100ee1a4f6d158b0)